website/docs/r/workflow.html.markdown

```diff
@@ -31,6 +31,7 @@ resource "newrelic_workflow" "foo" {
 
   destination {
     channel_id = newrelic_notification_channel.some_channel.id
+    notification_triggers = [ "ACTIVATED" ]
```

Comment:
minor: I would remove this part to keep the first example as small as possible, but add it to one of the other examples. But this is not critical at all.